src/cmd/mod.rs

```diff
@@ -210,11 +210,8 @@ pub fn load_wallet(
             GunSigner::SeedWordsFile {
                 file_path,
                 has_passphrase,
+                master_fingerprint,
```

Comment:
:bulb: It would be cool if we could change this so you only needed the fingerprint in the config if it has a password...

e.g. replace `has_passphrase` and `master_fingerprint` with
```
passphrase_fingerprint: Option<FIngerprint>
```

and use a `PwSeedSigner` if it's `Some`. WDYT?